website/blog/2022-08-18-new-release.md

```diff
@@ -0,0 +1,218 @@
+---
+title: What's Fresh in Databend v0.8
+description: Databend V0.8 Release Notes
+slug: databend-release-v0.8
+date: 2022-08-18
+tags: [databend, V0.8]
+authors:
+- name: Xuanwo
+  url: https://github.com/Xuanwo
+  image_url: https://github.com/Xuanwo.png
+---
+
+Hello, everyone! I'm Xuanwo. Today, on behalf of the Databend community, I would like to announce the official release of v0.8.
+
+Development of Databend v0.8 started on March 28th, with 5000+ commits and 4600+ file changes. In the last 5 months, the community of 120+ contributors added 420K lines of code and removed 160K lines, equivalent to rewriting Databend once. In this release, the community made significant improvements to the SQL Planner framework and migrated all SQL statements to the new Planner, providing full JOIN and subquery support.
+
```

Comment:
Insert the download link here.

Reply:
Could you provide the link?

Reply:
Release Note & Download Link: https://github.com/datafuselabs/databend/releases/tag/v0.8.0-nightly